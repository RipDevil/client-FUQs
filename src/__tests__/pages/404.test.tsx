import { render } from '@testing-library/react';
import PageNotFound from 'pages/404/page';

test('404 should be rendered', () => {
  const { container } = render(<PageNotFound />);
  expect(container).toMatchSnapshot();
});
