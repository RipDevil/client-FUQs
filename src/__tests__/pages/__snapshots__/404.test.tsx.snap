// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 should be rendered 1`] = `
.c0 {
  padding-top: 40vh;
  text-align: center;
}

<div>
  <div
    class="ant-row ant-row-center ant-row-middle c0"
  >
    <b>
      /404
    </b>
  </div>
</div>
`;
